timelock-corruption-detection/src/main/java/com/palantir/timelock/history/models/ConsolidatedLearnerAndAcceptorRecord.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.timelock.history.models;
+
+import java.util.Map;
+
+import org.immutables.value.Value;
+
+/**
+ * Rather than having two maps - one for learner records and one for acceptor records,
+ * the consolidated record has a sequence number mapped to pair of (learnedValue, acceptedValue).
```

Comment:
I think it's useful to explain _why_ the data is structured in this way as well, not just what it is